guides/gatherers.md

```diff
@@ -66,3 +67,30 @@ facts:
 ```
 
 For extra information refer to [trento-project/agent/../gatherers/corosyncconf_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/corosyncconf_test.go)
+
+
+### package_version
+
+This gatherer allows checking the installation of a certain package in the system
+
+Sample arguments
+| Name                   | Return value  
+| ---------------------- | -------------------------------------------------------------------------------------
+| `package_name`         | a string containing the installed version for the package `package_name`, e.g "2.4.5"
+
+Specification examples:
+
+```yaml
+facts:
+  - name: package_version-corosync
```

Comment:
Come on man! Let's avoid having this `_-` thins things in the examples! XD
Put underscore only
They are ugly as hell and people take reference on them

Reply:
It's not random! probably has a bit of OCD though :smile: .I want to make a clear difference between the name of the gatherer vs the argument we are giving

Reply:
I would still change it. It is obvous that we are talking about this gatherer as the `gatherer` field already says so.
I would be fine if you change to `corosync_version`.

Reply:
+1 to remove the dash and just use the lowdash for consistency. `corosync_version` or `corosync_package_version` will do

Reply:
@rtorrero I just realized it breaks the evaluation language as `-`  cannot appear in the variables: https://rhai.rs/book/ref/variables.html#valid-names